appveyor.yml

```diff
@@ -80,25 +79,31 @@ test_script:
             | Select-String -Pattern '<test-run id="[^"]*"\s+testcasecount="(\d+)"\s+result="([^"]*)"\s+total="(\d+)"\s+passed="(\d+)"\s+failed="(\d+)".*' `
             | foreach {
                 $result = $_.Matches.Groups
-                #$result[0].Value
+
                 $total  = $result[1].Value
                 $passed = $result[4].Value
                 $failed = $result[5].Value
                 $notPassed = $total - $passed
                 $testErrors = $failed
                 $testName = $resultFilePath.Split('\\')[-3]
                 $artifactName = "TestResult#${testRun}.${testErrors}err.${testName}"
-                Push-AppveyorArtifact $resultFilePath -FileName "${artifactName}.xml"
+
+                Write-Host "Pass#${testRun}: passed: $passed, failed: $failed, not passed: $notPassed, total: $total"
```

Comment:
```suggestion
                Write-Host "Pass #${testRun}: passed: $passed, failed: $failed, not passed: $notPassed, total: $total"
```

Reply:
These prints are for debugging purposes only, I'll remove them in the final version.
It took me awhile to realise test failures were ignore at the AppVeyor level... 🤦‍♂